lib/amazon-pricing/aws-price-list.rb

```diff
@@ -75,9 +90,9 @@ def self.fetch_url(url)
         # http://stackoverflow.com/questions/2060356/parsing-json-without-quoted-keys
         JSON.parse(body.gsub(/(\w+)\s*:/, '"\1":'))
       end
-    rescue
-      $stderr.puts "Failed to parse: #{url}"
-      raise
+    rescue StandardError => e  #Exception => e
```

Comment:
it's good to keep parity, but I think the initial attempt was to rescue all exceptions so this could be Exception => e